README.md

```diff
@@ -19,92 +24,98 @@ Capture and analyze social media for a specified Twitter handle or hashtag and l
+* Once you have the Twitter account created and verified, log in to [Twitter Dev](https://apps.twitter.com/) and create an application.  
+* Select the Keys and Access Tokens tab and generate a Consumer Key and Secret.
+Keep this page open as you will need to use these tokens into setup procedure in the application later on.
 
-Once you have the Twitter account created and verified, log in to [Twitter Dev](https://apps.twitter.com/) and create an application.  Select the Keys and Access Tokens tab and generate a Consumer Key and Secret.
+### 3. Create Watson services with IBM Bluemix
 
-Keep this page open as you will need to use these tokens into setup procedure in the application later on.
+Either Setup the Bluemix Deployment or Setup Local Deployment
 
-## Setting up Bluemix
+#### Setting up Bluemix Deployment
 
 > Explanation: You will create a placeholder application in Bluemix that connects to all the required services first.
 
-1. If you do not already have a Bluemix account, [sign up here](https://console.ng.bluemix.net/registration).
+1. If you do not already have a Bluemix account, [sign up for Bluemix](https://console.ng.bluemix.net/registration).
 2. Download and install the [Cloud Foundry CLI](https://console.ng.bluemix.net/docs/cli/index.html#cli) tool.
-3. Log into Bluemix with our account.
-4. From the Application Dashboard, Create a new Application.
-  - On the left, select Apps > Cloudfoundry Apps.
-  - On the right, select SDK for Node.js.
+3. Log into Bluemix with your account.
+4. From the `Application Dashboard`, Create a new `Application`.
+  - On the left, select `Apps > Cloudfoundry Apps`.
+  - On the right, select `SDK for Node.js`.
   - Provide a unique name for your application.
-5. Once the application is created, go into the application and select Connections.
-6. Create the required services and bind them to the newly created application.
-7. Leave the Connections page open, as you will reference the credentials in the subsequent setup step.
+5. Once the application is created, go into the application and select `Connections`.
+6. Create the required services and bind them to the newly created application: `Watson Conversation`, `Natural Lanuguage Understanding`, `Tone Analyzer`, and `Cloudant NoSQL DB`.
+7. Leave the `Connections` page open, as you will reference the credentials in the subsequent setup step.
+
+#### Setup local Deployment
 
-## Load/Create the Conversation API workspaces
+> Explanation: You will create the  Bluemix services and configure them to use on a locally running server app.
 
-> Note: Although we do provide you with a sample classification workspace and dialog workspace.  The idea is that you create your own that is relevant to your domain.
+If you do not already have a Bluemix account, [sign up for Bluemix](https://console.ng.bluemix.net/registration).
```

Comment:
region-specific links here and L105-L108

Reply:
done